account_invoice_supplier_ref_unique/tests/test_account_invoice_supplier_ref_unique.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Acsone
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp.exceptions import ValidationError
+from openerp.tests.common import SavepointCase
+
+
+class TestAccountInvoiceSupplierRefUnique(SavepointCase):
+
+    @classmethod
+    def setUpClass(self):
+        super(TestAccountInvoiceSupplierRefUnique, self).setUpClass()
+
+        # ENVIRONMENTS
+        self.account_account = self.env['account.account']
+        self.account_invoice = self.env['account.invoice']
+
+        # INSTANCES
+        self.partner = self.env.ref('base.res_partner_2')
```

Comment:
I prefer to not use demo data, as they change from version to version, and this case is very simple to re-create a new one.